packages/core/src/components/pagination/pagination.tsx

```diff
@@ -1,13 +1,28 @@
-import { h, Component, ComponentInterface, Event, EventEmitter, Prop } from '@stencil/core';
+import { h, Component, ComponentInterface, Element, Event, EventEmitter, Fragment, Prop, State } from '@stencil/core';
 
 import { PaginationSelectPageEvent } from './pagination.interfaces';
 
+export type LabelSizeMap = { [key in string]: number; }
```

Comment:
Is er nog een type beschikbaar voor `'small' | 'medium' | 'large'` dan kan die hier ingezet worden.

Daarnaast denk ik dat ik `Record<string, number>` een prettigere notatie vind, wat denk jij?

Reply:
Het kan allebei, in geval van `Record<string, number>` hoeft er geen custom type gemaakt te worden, dus lijkt mij dat beter.

Reply:
Goed plan. Ook beter voor als er ooit nog een maat wordt toegevoegd.